kura/org.eclipse.kura.core.system/META-INF/MANIFEST.MF

```diff
@@ -12,7 +12,7 @@ Import-Package:
  org.eclipse.kura;version="[1.0,2.0)",
  org.eclipse.kura.executor;version="[1.0,2.0)",
  org.eclipse.kura.net;version="[2.0,3.0)",
- org.eclipse.kura.system;version="[1.5,1.6)",
+ org.eclipse.kura.system;version="[1.5,2.0)",
```

Comment:
I think this implements the system apis and should be [1.5,1.7). Maybe have a look at this to understand better the versioning scheme adopted by Kura: https://github.com/eclipse/kura/wiki/Kura-Semantic-Versioning